modules/Meetings/Meeting.php

```diff
@@ -546,9 +546,9 @@ function get_list_view_data() {
 			//cn: added this if() to deal with sequential Closes in Meetings.	this is a hack to a hack(formbase.php->handleRedirect)
 			if(empty($action))
 			     $action = "index";
-            $setCompleteUrl = "<a id='{$this->id}' class='list-view-data-icon' title='".translate('LBL_CLOSEINLINE')."' onclick='SUGAR.util.closeActivityPanel.show(\"{$this->module_dir}\",\"{$this->id}\",\"Held\",\"listview\",\"1\");'>";
-			if ($this->ACLAccess('edit')) {
-                $meeting_fields['SET_COMPLETE'] = $setCompleteUrl . "<span class='suitepicon suitepicon-action-clear'></span></a>";
+            $setCompleteUrl = "<b><a id='{$this->id}' class='list-view-data-icon' title='".translate('LBL_CLOSEINLINE')."' onclick='SUGAR.util.closeActivityPanel.show(\"{$this->module_dir}\",\"{$this->id}\",\"Held\",\"listview\",\"1\");'>";
```

Comment:
@e-reeley Why b tag though? Is this consistent with other a href in list views? I see we have this list-view in Calls aswell, are not looking at that also?

Reply:
@samus-aran Here ive used the b tag to override the style which seems to be applied to most columns within table which underlines the populated values. This applies a different style to the field. I could search another fix if this way is not most efficient? 

Ill also have a look and alter the other list views that I have missed and add them into this pr. Thanks!